apps/health_campaign_field_worker_app/lib/pages/inventory/record_stock/stock_details.dart

```diff
@@ -459,7 +459,19 @@ class _StockDetailsPageState extends LocalizedState<StockDetailsPage> {
                                 ),
                                 menuItems: reasons ?? [],
                                 formControlName: _transactionReasonKey,
-                                valueMapper: (value) => value.name.titleCase,
+                                valueMapper: (value) {
+                                  if (value.name.titleCase ==
+                                      "Lost In Storage") {
+                                    return localizations.translate(
+                                        i18.stockDetails.lostInStorage,);
+                                  } else if (value.name.titleCase ==
+                                      "Lost In Transit") {
+                                    return localizations.translate(
+                                        i18.stockDetails.lostInTransit,);
+                                  } else {
+                                    return value.name.titleCase;
+                                  }
+                                },
```

Comment:
directly use localizations.translate(value.title,) instead of if else conditions and add localization against that value LOST_IN_STORAGE etc.